broker/src/main/java/io/camunda/zeebe/broker/system/partitions/impl/AsyncSnapshotDirector.java

```diff
@@ -256,6 +259,12 @@ private void takeSnapshot() {
           if (snapshotTakenError != null) {
             if (snapshotTakenError instanceof SnapshotException.SnapshotAlreadyExistsException) {
               LOG.debug("Did not take a snapshot. {}", snapshotTakenError.getMessage());
+            } else if (snapshotTakenError instanceof NoEntryAtSnapshotPosition
```

Comment:
:wrench: this is getting a bit unwieldy I think, maybe we could have a `logSnapshotTakenError` method?